### PR TITLE
Rationalize components

### PR DESCRIPTION
**Pull request by jnewbery**
...so that they match the labels in github.

### CODE REVIEW

File: _posts/2019-05-01-#15557.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: pr
 title: "Enhance bumpfee to include inputs when targeting a feerate"
-components: [wallet]
+components: ["wallet"]
```
Comment (jonatack) [inline _posts/2019-05-01-#15557.md:4]: https://github.com/bitcoin/bitcoin/pull/15557 seems to be tagged "tx fees and policy"

**Comment by jonatack** [comment]
It looks like the GitHub labels in Bitcoin Core have been updated a bit. Opening a PR to update them here.

File: _posts/2019-09-11-#16688.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: pr
 title: "Add validation interface logging"
-components: [logging]
+components: ["utils/log/libs"]
```
Comment (jonatack) [inline _posts/2019-09-11-#16688.md:4]: ["utils/log/libs", "validation"]

**Comment by jonatack** [comment]
Cross-checked with the changed labels in Bitcoin Core shown below, and with the above comments I think we're good. Thanks!

```diff
 DESIRED_COMPONENTS = [
+  'Block storage',
   'Build system',
-  'Config',
   'Consensus',
   'Descriptors',
   'Docs',
   'GUI',
-  'Logging',
   'Mempool',
   'Mining',
-  'Net processing',
   'P2P',
-  'Policy',
+  'Privacy',
```

File: _posts/2019-05-01-#15557.md (hunk 1 above)
Comment (jnewbery) [reply _posts/2019-05-01-#15557.md:4]: No, this is definitely a wallet PR

File: _posts/2019-05-08-#10823.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: pr
 title: "Allow all mempool txs to be replaced after a configurable timeout (default 6h)"
-components: [mempool, policy]
+components: ["mempool"]
```
[the remark this answers is not in the document]
Comment (jnewbery) [reply _posts/2019-05-08-#10823.md:4]: This is a mempool PR. I'm not sure if policy makes sense for us as a separate label

File: _posts/2019-09-11-#16688.md (hunk 2 above)
Comment (jnewbery) [reply _posts/2019-09-11-#16688.md:4]: This PR doesn't change anything in validation.

**Comment by jnewbery** [comment]
Thanks for the review, @jonatack . We don't need to slavishly follow the labels in the Bitcoin Core repo. Those are set by one person so they're bound to be subjective to a certain extent. It's definitely useful to scrape that label when creating the PR review club page, but we can change the component afterwards to make it more meaningful for us.

**Comment by jonatack** [comment]
I agree -- it's more useful to label PRs by what they really do, not based on the files they may touch or the labels they were given.

**Pull request merged**